core/xchain/server/server.go

```diff
@@ -187,7 +186,6 @@ func New(
 	}
```

Comment:
Is the above setting being properly applied here now that chain monitor start happens on blockchain creation?

Reply:
no, I'll fix it...